jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -349,16 +349,7 @@ public void invoke(final long window, final boolean focus) {
 
             @Override
             public void invoke(final long window, final int width, final int height) {
-
-                // This is the window size, never to passed to any pixel based stuff!
-                // https://www.glfw.org/docs/latest/window_guide.html#window_size
-                onWindowSizeChanged(width, height);
-
-                // Notify listeners
-                for (WindowSizeListener listener : windowSizeListeners.getArray()) {
-                    listener.onWindowSizeChanged(width, height);
-                }
```

Comment:
```
// Notify listeners
for (WindowSizeListener listener : windowSizeListeners.getArray()) {
    listener.onWindowSizeChanged(width, height);
}
```

Is this removed by mistake?

Reply:
Good catch! I was supposed to move that loop into `updateSizes()` but somehow it got dropped. I'll fix.